common-react/@dbeaver/ui-kit/src/Select/Select.css

```diff
@@ -80,6 +83,9 @@
     cursor: pointer;
     border: none;
     background: none;
+    overflow: hidden;
+    white-space: nowrap;
+    text-overflow: ellipsis;
```

Comment:
why do we need this changes in the basic Select implementation? Multi-row items are pretty standard, we might want to have it somewhere at some time. We can add this styles to cloudbeaver combobox.

<img width="275" alt="image" src="https://github.com/user-attachments/assets/251f8848-7540-414f-8d8c-c0ce8aa82c20" />